src/kvirc/ui/KviWindow.cpp

```diff
@@ -320,14 +320,13 @@ const char * KviWindow::m_typeTable[TypeCount] = {
 	"userwindow",   // 16
 	"tool",         // 17
 	"iograph",      // 18
-	"dirbrowser",   // 19
```

Comment:
Please don't delete this and move down the names, it will mess up the order of the array since the names are in a specific order (similar to the config ordeal)